examples/auth0/LICENSE

```diff
@@ -0,0 +1,19 @@
+The MIT License
```

Comment:
Do we need this license file? None of our other examples are licensed. I think this may deter usage.

Reply:
Removed we also have one in `with-firebase-hosting-and-docker` should we remove that one also?